symbology/simple-fill-symbol/src/main/java/com/esri/samples/simple_fill_symbol/SimpleFillSymbolSample.java

```diff
@@ -80,17 +85,18 @@ public void start(Stage stage) {
 
       createSymbolFunctionality();
 
-      final ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
-
-      // set initial map view point
-      Point initialPoint = new Point(-12000000, 5400000, SpatialReferences.getWebMercator());
-      Viewpoint viewpoint = new Viewpoint(initialPoint, 10000000); // point, scale
-      map.setInitialViewpoint(viewpoint);
+      // create a map with the topographic basemap style
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
-      // create a view for this ArcGISMap and set ArcGISMap to it
+      // create a map view and set its map
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      Point initialPoint = new Point(-12000000, 5400000, SpatialReferences.getWebMercator());
+      Viewpoint viewpoint = new Viewpoint(initialPoint, 10000000); // point, scale
```

Comment:
We could probably get rid of this `initialPoint` variable and just put it straight in the `Viewpoint` constructor.
Either way, we could definitely get rid of the comment at the back of the line creating the `Viewpoint` which describes the input parameters.
```suggestion
      Viewpoint viewpoint = new Viewpoint(new Point(-12000000, 5400000, SpatialReferences.getWebMercator()), 10000000);
```

Reply:
Have removed the `Point` and just set the `Viewpoint`.